nixos/modules/services/misc/redmine.nix

```
@@ -74,6 +74,12 @@ in
         description = "Group under which Redmine is ran.";
       };
 
+      address = mkOption {
+        type = types.str;
+        default = "0.0.0.0";
```

```suggestion
        default = "localhost";
```

To keep default according to https://guides.rubyonrails.org/command_line.html#bin-rails-server

---

My Redmine instance doesn't have that parameter and it's listening on 0.0.0.0. No idea why that is the case but I trust my Redmine instance more rather than this guide.

---

It might be only on your instance.
I did look at redmine source, and found:
```
$ grep localhost config/settings.yml 
  default: localhost:3000
```
Do you have any settings which might override this somewhere ?

---

https://www.redmine.org/projects/redmine/wiki/redminesettings#Host-name-and-path

---

This setting has nothing to do with the socket. I also set up new instances with the same behaviour. I'm just using the Redmine NixOS Module.

---

It might be that 0.0.0.0 is specific to Redmine https://www.redmine.org/boards/2/topics/13220

---

Hum, I don't have a "resolve" button :/

---

Done. Thank you!